BigriverBookstore-apple/Sources/New Group/DetailsPhotoItem/CollectionViewDetailsPhotoItem.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CollectionViewDetailsPhotoItem.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 07/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import Cocoa
+
+class CollectionViewDetailsPhotoItem: NSCollectionViewItem {
+    @IBOutlet var currentImageView: NSImageView!
+    weak var delegate: CollectionViewDetailsPhotoItemDelegate?
+    var photoInfo: Photo? {
+        didSet {
+            if let photoI = photoInfo {
```

Comment:
@c-Viorel Maybe with guard and better variable name for 'photoI' this method will look better.